staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -81,6 +81,11 @@ type ListMeta struct {
 	// identical to the value in the first response, unless you have received this token from an error
 	// message.
 	Continue string `json:"continue,omitempty" protobuf:"bytes,3,opt,name=continue"`
+
+	// RemainingItemCount is the number of remaining items if the list request does not contain label
```

Comment:
I suggest:

RemainingItemCount is the number of subsequent items in the list which are not included in this list response. If the list request contained label or field selectors, then the number of remaining items is unknown and this field will be unset. If the list is complete, then this field will also be unset.

Reply:
Applied the suggestion. Thanks.